tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -125,79 +125,79 @@
 // Delegate for testing Cortana foreground activation
 // Use text value to guarantee it is the same argument we create
 @interface CortanaVoiceCommandForegroundTestDelegate : NSObject <UIApplicationDelegate>
-@property (nonatomic, readonly) NSMutableDictionary* methodsCalled;
+@property (nonatomic, readonly) StrongId<NSMutableDictionary> methodsCalled;
```

Comment:
But why? I am not seeing what this is actually addressing.  

---
In reply to: [82297024](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82297024) [](ancestors = 82297024,82296469)